sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala

```diff
@@ -388,14 +388,15 @@ private[ui] class ExecutionPagedTable(
         +details
       </span> ++
       <div class="stage-details collapsed">
-        <pre>{execution.details}</pre>
+        <pre>{execution.description}<br></br>{execution.details}</pre>
```

Comment:
If there is no truncation as default, we can ignore the duplicated `execution.description` here.

Reply:
@gengliangwang that may be hard to do. For example a query may not be truncated but even a browser window resize event, css will truncate the query as per new window size without a server call right.? how to handle then.?

Reply:
@gengliangwang i hope you are talking about css truncation and not the server side truncation.. right.?

Reply:
I see. No problem here :)